src/patternfly/layouts/Stack/examples/stack-example.hbs

```diff
@@ -1,8 +1,8 @@
-{{#> stack classModifier="Documentation-container-xtall"}}
+{{#> stack stack--modifier="Documentation-container-xtall"}}
```

Comment:
again doesn't have to be part of this PR, but if possible we should remove this workspace class and see if we can get the demo to work modifying `index.js`

Reply:
ah, ya I made that comment on the level layout. We can just create an issue for a fix for this if we don't want to fix it now.